browser/components/CodeEditor.js

```diff
@@ -50,7 +50,7 @@ export default class CodeEditor extends React.Component {
     this.value = this.props.value
     this.editor = CodeMirror(this.refs.root, {
       value: this.props.value,
-      lineNumbers: true,
+      lineNumbers: this.props.lineNumber,
```

Comment:
I prefer `lineNumbers` as the variable name.

Reply:
Yeah, I prefer `lineNumbers` too. I used `lineNumber` as it was already used somewhere else in the code.

Reply:
> it was already used somewhere else in the code.

Where is it? I could not find it... Does it mean you chose `lineNumber` because there are probabilities to conflict?

Reply:
It's used for the line numbers in the preview. [Here](https://github.com/BoostIO/Boostnote/blob/95d74d1ca2206ed18add0348682af0063a5ac5b0/browser/main/lib/ConfigManager.js#L45), for example.

Reply:
I see, and the name is also not good... I'll have to change it sooner or later.

Reply:
So should I change it to `lineNumbers`?